docs/configuration.md

```diff
@@ -892,6 +892,15 @@ _sqlDialect=sqlite only_
 
 The path to the sqlite database on the local filesystem relative to the current working directory.
 
+##### `storage.sqlSocketPath`
+
+_sqlDialect=mysql or sqlDialect=postgres only_
+
+The database Unix socket file path for the MySQL or PostgreSQL database.
+
+Note: If used, this option will require to specify the `database`, `username` and `password` sequelize options.
```

Comment:
A complete example configuration in the examples section would be nice to link to :)